### ISSUE TITLE
Organize imports always adds one more blank line after use statements before class

### ISSUE THREAD

**Issue by still-dreaming-1** [DEVSENSE/phptools-docs#739]
VSCode extension v1.54.16480

Here are my settings:
```
        "[php]": {
            "files.trimTrailingWhitespace": true,
            "editor.tabSize": 4,
            "editor.codeActionsOnSave": {
                "source.organizeImports": "explicit",
            },
        },
        "php.version": "8.1",
        "php.completion.autoimport": "auto-import",
        "php.completion.autoimport-docblock": "auto-import",
        "php.inlayHints.parameters.byReference": true,
        "php.inlayHints.parameters.enabled": true,
        "php.inlayHints.parameters.suppressNameMatchingValue": true,
        "php.format.rules.arrayInitializersNewLineAfterLastElement": true,
        "php.format.rules.arrayInitializersNewLineBeforeFirstElement": true,
        "php.format.rules.arrayInitializersWrap": "onEveryItem",
        "php.format.rules.callParametersNewLineAfterLeftParen": true,
        "php.format.rules.callParametersNewLineBeforeRightParen": true,
        "php.format.rules.callParametersWrap": "onEveryItem",
        "php.format.rules.catchOnNewLine": false,
        "php.format.rules.declKeepRightParenAndOpenBraceOnOneLine": true,
        "php.format.rules.declParametersNewLineAfterLeftParen": true,
        "php.format.rules.declParametersNewLineBeforeRightParen": true,
        "php.format.rules.forStatementNewLineAfterLeftParen": true,
        "php.format.rules.forStatementNewLineBeforeRightParen": true,
        "php.format.rules.forStatementWrap": "onEveryItem",
        "php.format.rules.groupUseWrap": "onEveryItem",
        "php.format.rules.ifStatementNewLineAfterLeftParen": true,
        "php.format.rules.ifStatementNewLineBeforeRightParen": true,
        "php.format.rules.implementsListWrap": "onEveryItem",
        "php.format.rules.newLineAfterImplements": true,
        "php.format.rules.openBraceOnNewLineForFunctions": false,
        "php.format.rules.openBraceOnNewLineForNamespaces": true,
        "php.format.rules.openBraceOnNewLineForTypes": true,
        "php.format.rules.spaceAroundConcatenation": true,
        "php.format.rules.spaceBeforeColonInReturnType": false,
        "php.format.rules.spaceBeforeParenthesesInControlStatements": true,
        "php.format.rules.spaceAfterCast": false,
        "php.format.rules.switchStatementNewLineAfterLeftParen": true,
        "php.format.rules.switchStatementNewLineBeforeRightParen": true,
        "php.format.rules.whileStatementNewLineAfterLeftParen": true,
        "php.format.rules.whileStatementNewLineBeforeRightParen": true,
        "php.format.rules.addCommaAfterLastArrayElement": true,
        "php.format.rules.addCommaAfterLastCallParameter": true,
        "php.format.rules.addCommaAfterLastDeclParameter": true,
        "php.format.rules.blankLinesBeforeClass": 1,
        "php.format.rules.blankLinesAfterClass": 0,
        "php.format.rules.blankLinesBeforeNamespace": 1,
        "php.format.rules.blankLinesAfterNamespace": 1,
        "php.format.rules.blankLinesBeforeUseStatements": 1,
        "php.format.rules.blankLinesAfterUseStatements": 1,
        "php.format.rules.blankLinesAfterMethod": 0,
        "php.format.rules.blankLinesAfterFunction": 0,
        "php.format.rules.blankLinesBeforeFunction": 1,
        "php.format.rules.blankLinesBeforeFunctionBody": 0,
        "php.format.rules.blankLinesBeforeMethod": 1,
        "php.format.rules.blankLinesBeforeMethodBody": 0,
        "php.format.rules.maxBlankLines": 1,
        "php.format.rules.booleanConstantCasing": "lowercase",
        "php.format.rules.nullConstantCasing": "lowercase",
        "php.format.rules.endWithNewLine": true,
```
After the last update, whenever I save, it adds one more blank line after use statements before the class (I have it configured to organize imports upon saving). So if there were no blank lines between them, it will add one. If I then save again, it will another, and every time I save again, it will add one more. I updated all the settings I could find related to blank lines to be as desired, and the problem still exists. The above settings are my updated ones. I also tried changing both the blank lines after use statements to 0 and the blank lines before class to 0 with no impact.

Rolling back to v1.53.16379 resolves the issue.

**Comment by jakubmisek** [comment]
Thank you for reporting that.

I'm not able to replicate the issue with these settings though. Is it happening with any code?

**Comment by still-dreaming-1** [comment]
> "source.organizeImports": "explicit

It seems to yes. I reproduced it on a couple different files because one was a bit odd so I wanted to see if it would still happen in a simpler class file. Here is an example of similar code to what I tried it on:

```
<?php

namespace This\Name\Space;

use Some\Namespace\OtherClass;

final class ThisClass
{
}

```

The number of blank lines between the use statements and the start of the class increases every time I save.

**Comment by jakubmisek** [comment]
I see, thanks @still-dreaming-1 !

Fixed, tested, and preparing update.

**Comment by still-dreaming-1** [comment]
I can confirm it is fixed for me in the latest pre-release. Thanks.

**Comment by jakubmisek** [comment]
@still-dreaming-1 thank you for the confirmation

**Comment by taisph** [comment]
This one was driving me nuts to figure out. I can confirm it is fixed in the 1.54.16521 pre-release.

**Comment by jakubmisek** [comment]
@taisph thank you for confirmation, we'll publish the full release soon.
